components/molecules/PopularCategoryCard.js

```diff
@@ -66,4 +68,16 @@ PopularCategoryCard.propTypes = {
    * Callback for a click event on the card
    */
   onClick: PropTypes.func,
+
+  /**
+   * Callback for a click event on the card
+   */
+  selected: PropTypes.bool.isRequired,
+};
+
+PopularCategoryCard.defaultProps = {
+  /**
+   * Default value of false for selected
+   */
+  selected: false,
```

Comment:
Interesting, I didn't know about defaultProps. If selected is required is a default needed? Also booleans are false by default